expertise/utils/utils.py

```diff
@@ -1,5 +1,6 @@
 from __future__ import print_function, absolute_import, unicode_literals
 import codecs
+import shutil
```

Comment:
Looks like these changes are leftover from my original code, you can remove them to make the branch cleaner